### ISSUE TITLE
Setting group image converts to JPEG with black background

### ISSUE THREAD

**Issue by rosano** [deltachat/deltachat-desktop#3037]
- Operating System (Linux/Mac/Windows/iOS/Android): macOS
- Delta Chat Version: 1.29.0
- Expected behavior: Image maintains transparency or uses white background
- Actual behavior: Image loses transparency and/or uses black background
- Steps to reproduce the problem: Upload a group image

Below is a sample image that can be used for testing. Because it's only a line drawing it ends up being completely black.

![test](https://user-images.githubusercontent.com/1680612/204473056-57f88f1c-b0b4-4970-9f98-a96650bec5e9.png)

**Comment by r10s** [comment]
> Expected behavior: Image maintains transparency or uses white background

maintaining transparency should be done only as a workaround if it is the simpler solution "for now". transparency, shapes, alpha in user-avatars and webxdc-icons is a non-goal, resulting easily in lots of issues.

using a defined background color seems to be the better solution - and even if one could also have a png with only white lines, defaulting to a white background probably solves the majority of cases in practice.

**Comment by rosano** [comment]
> defaulting to a white background probably solves the majority of cases in practice.

Yes I was basically suggesting this, as I have used images from the web several times for group discussions, and each time I need to convert it to jpeg in order for it to appear correctly. Seems like it would be simple to solve.

> using a defined background color seems to be the better solution

There could be a user defined colour but the app converts the image to jpeg with a black background, so any CSS styling would be overlapped.

**Comment by Simon-Laux** [comment]
in desktop this is done by core, though on iOS the background is set to white instead of black, so maybe iOS has different logic for it?
